Pipfile

```diff
@@ -9,12 +9,13 @@ pipenv = "*"
 requests = "*"
 toml = "*"
 "delegator.py" = "*"
-gitpython = "==1.0.1"
+gitpython = "==2.1.8"
 pyyaml = "*"
 semver = "*"
 thoth-common = "*"
 thamos = "*"
 ogr = "*"
+gitdb2 = "==3.0.1"
```

Comment:
Do we need this to be pinned?

Reply:
That's what they suggest as after that it breaks.

Reply:
https://github.com/gitpython-developers/GitPython/issues/983
Alternative is to use latest gitpython version which I have made another PR for, we can close this if we use the latest version.

Reply:
Let's use the more recent version then.